src/object/object_domain.c

```diff
@@ -10187,6 +10188,20 @@ tp_value_cast (const DB_VALUE * src, DB_VALUE * dest, const TP_DOMAIN * desired_
   return tp_value_cast_internal (src, dest, desired_domain, mode, true, false);
 }
 
+TP_DOMAIN_STATUS
+tp_value_cast_ex (const DB_VALUE * src, DB_VALUE * dest, const TP_DOMAIN * desired_domain, bool implicit_coercion,
```

Comment:
how about tp_value_cast_force ()? it seems that bool force parameter is only passed true.